### PR TITLE
update urls

### PR DESCRIPTION
**Pull request by mbodeantor**
We've added a /api to the api url since this was written

### CODE REVIEW

**Review by kalenluciano — APPROVED** [review]
Good catch changing this!

**Comment by mbodeantor** [comment]
@josh-chamberlain just need your approval as code owner

**Review by josh-chamberlain — APPROVED** [review]

**Pull request merged**
